Collect and display the page title for a department's content

leenagupte opened the pull request:
Trello card: https://trello.com/c/q721ivrT

## Description

Get the titles of every piece of content that belongs to an organisation.

## Preview

![screen shot 2016-12-02 at 11 39 18](https://cloud.githubusercontent.com/assets/5793815/20832763/03b61872-b884-11e6-881a-2520e5c115bf.png)

pmanrubia commented on app/models/importers/organisation.rb:
```diff
@@ -2,6 +2,8 @@ class Importers::Organisation
   attr_reader :slug, :batch, :start
   attr_writer :start
 
+  FIELDS = %w(content_id title)
```
The name `FIELDS` is not very descriptive. Can you think of a better name?

pmanrubia commented on app/models/importers/organisation.rb:
```diff
@@ -13,7 +15,7 @@ def run
       result = search_content_items_for_organisation
       result.each do |content_item_attributes|
-        attributes = content_item_attributes.slice('content_id', 'title')
+        attributes = content_item_attributes.slice(*FIELDS)
```
I like how you used the `splat` operator here 👍

leenagupte commented on spec/models/importers/organisation_spec.rb:
```diff
@@ -1,8 +1,8 @@
 require 'rails_helper'
```
Will do.

leenagupte commented:
@pmanrubia I have moved the typo fix to the first commit.

I've moved updating the model (schema?) to the commit where the importer is updated, the first place that the `title` column is needed.

pmanrubia commented:
LGTM
Please, remember to move the constant below the private section (along with `private_const`) as does not describe the interface of the class.

leenagupte commented:
@pmanrubia I've moved the private_constant as requested, and rebased. Merging...
